src/ontology/mondo-edit.obo

```diff
@@ -293933,20 +293913,36 @@ property_value: confidence "0.7021739130434781" xsd:double
 
 [Term]
 id: MONDO:0014700
-name: neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation
+name: Au-Klein syndrome
```

Comment:
Based on https://www.omim.org/entry/616580 should the name be `Au-Kline syndrome` vs. `Klein`?

Reply:
good catch! I will fix this